examples/ES/noise.py

```diff
@@ -0,0 +1,33 @@
+# Third party code
+#
+# The following code are copied or modified from:
+# https://github.com/ray-project/ray/blob/master/python/ray/rllib/utils/filter.py
+
+import numpy as np
+
+
+class SharedNoiseTable(object):
+    """
```

Comment:
Pls add more comment to describe its definition and usage.

Reply:
done